examples/remote-file.js

```diff
@@ -0,0 +1,5 @@
+const { imageHash } = require('../dist');
+
+imageHash('https://ichef-1.bbci.co.uk/news/660/cpsprodpb/7F76/production/_95703623_mediaitem95703620.jpg', 16, false).then(res => {
```

Comment:
This should really use mocked service worker or something to prevent the need for the actual remote image.